analyses/data-pre-release-qc/results/fusion-starfusion-samples-missing-in-histologies.tsv

```diff
@@ -1,5 +1,6 @@
 Kids_First_Biospecimen_ID
 TARGET-50-CAAAAQ-11A-01R
+BS_JXF8A2A6
 BS_4PRD2E3Q
 BS_BXSX4A7H
 BS_47EYP04A
```

Comment:
can you check you have the current files- BS_JXF8A2A6 for ex should be inactive in the master genomics filesheet and should not be used, should not be in our files. Looks like BS_4PRD2E3Q is a normal as well as BS_4PRD2E3Q, and BS_47EYP04A although I am having doubts these are annotated properly and probably should be tumor.

Reply:
I added these 3 for Jenn Mason to inspect [here](https://docs.google.com/spreadsheets/d/196j90wXCF37vx_hliltEYX30az3J4jTct4usObf-bsA/edit#gid=0)